raftinc/allocate.hpp

```diff
@@ -2,14 +2,14 @@
  * allocate.hpp - allocate base class, extend me to build new
  * allocate classes.  This object has several useful features
  * and data structures, namely the set of all source kernels and
- * all the kernels within the graph.  There is also a list of 
+ * all the kernels within the graph.  There is also a list of
  * all the currently allocated FIFO objects within the streaming
  * graph.  This is primarily for instrumentation puposes.
  * @author: Jonathan Beard
  * @version: Tue Sep 16 20:20:06 2014
- * 
+ *
```

Comment:
@BamboWu , did UT (or yourself) want to also add yourself to the author line?

Reply:
This one should be fine, I think.
The most of modifications I did is on another branch [armq_dev](https://github.com/UT-LCA/RaftLib/tree/armq_dev), and I added UT in the header there.